mlir/docs/Tutorials/transform/Ch1.md

```diff
@@ -11,6 +11,9 @@ The application of transform IR always starts from one top-level operation. In t
 Let us illustrate this with a simple sequence of transformations on the common “fully connected + bias + ReLU” ML layer, which boils down to performing a matrix multiplication, followed by an (elementwise) matrix addition and taking an elementwise maximum with 0. This can be expressed using the following IR:
 
 ```mlir
+#map0 = affine_map<(d0, d1) -> (d0, d1)>
+#map1 = affine_map<(d0, d1) -> ()>
```

Comment:
Nit: could you make these maps inlined? You can achieve that with the `-mlir-print-local-scope`. I find these "forward declared" maps annoying because you have to either recall what they are or constantly go back to the start of the file...